addon/components/bs4/bs-form/element/help-text.js

```diff
@@ -0,0 +1,5 @@
+import FormElementHelpText from 'ember-bootstrap/components/base/bs-form/element/help-text';
+
+export default FormElementHelpText.extend({
+  classNames: ['form-text']
```

Comment:
Thanks. Looks like I missed the documentation reference. Would you mind cleaning up the docs in that area?